scripts/check-cocoapods-locks.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Early exit when script is run on platforms other than macOS
+if [[ "$OSTYPE" != "darwin"* ]]; then
+  exit 0
+fi
+
+green="tput setaf 2"
+yellow="tput setaf 3"
+blue="tput setaf 4"
+reset="tput sgr0"
+
+HEADER_PRINTED=false
+
+check-cocoapods-locks() {
+  podfileLockPath="$1/Podfile.lock"
+  manifestLockPath="$1/Pods/Manifest.lock"
+  podfileLock=`md5 -q $podfileLockPath 2>&1`
+  manifestLock=`md5 -q $manifestLockPath 2>&1`
+
+  if [ "$podfileLock" != "$manifestLock" ]; then
+    if [ $HEADER_PRINTED = false ]; then
+      printf "\\n⚠️  $($yellow)Your local pods are outdated, please run $($blue)pod install$($yellow) in the following directories:$($reset)\\n"
```

Comment:
I think this message is going to get displayed a lot. Every time someone changes branches with different Pods we're going to show it, even if they aren't working on iOS.

The code would need to be moved around but I think it would be nice if we could print the message on one line so that it doesn't get in the way. I would also use a different word than "outdated" because sometimes the local pods can be newer than the ones in the repo (ex: checking out an old commit).

Alternatively, we could add check-cocoapods-locks to .envrc in the iOS directories instead of Git hooks.